### ISSUE TITLE
First Color Always Transparent

### ISSUE THREAD

**Issue by opiter09** [Garhoogin/NitroPaint#5]
As these pictures illustrate, the first color of a palette is always treated as transparent, even when it is clearly being used by the image. In the case of LEGO Battles, this seems to be resolvable by just only making the first color transparent if it is good old 255-0-255, but of course that might not work for every game. Maybe there could be a switch of some sort to let the user decide whether the first color needs to be transparent or not?
![Screenshot (454)](https://user-images.githubusercontent.com/92452568/162645367-bb0815d9-a32d-4add-aacb-d023a62fd688.png)
![Screenshot (455)](https://user-images.githubusercontent.com/92452568/162645510-0fd11118-94ab-4965-8e67-f85a6d41d49d.png)

**Comment by Garhoogin** [comment]
A couple of versions ago I encountered some other games  that used the first color in the palette for something non-transparent (think it was Time Ace), so I did include an option to view those graphics with the first color not treated as transparent, it should be in under View->Render Transparency. It can be selected from any context, but only applies to graphics for the 2D engine. Hope this works for you.
